include/openthread/instance.h

```diff
@@ -53,7 +53,7 @@ extern "C" {
  * @note This number versions both OpenThread platform and user APIs.
  *
  */
-#define OPENTHREAD_API_VERSION (22)
+#define OPENTHREAD_API_VERSION (23)
```

Comment:
submitted https://github.com/openthread/openthread/pull/5370 to avoid this change.

Reply:
Done